templates/vue/src/assets/styles/tyde-colors.scss

```diff
@@ -0,0 +1 @@
+$tyde-orange: #f79621;
```

Comment:
I think if we're going to move to `scss` variables the other `css` variables should be moved to `scss` as well.

Reply:
We should create a separate PR once this is merged in to update all CSS variables to SCSS.

Reply:
Sure. I am planning to open another PR for that (since there are quite a few places where we need to move these css variables). I just want to keep this PR as clean and separate as possible.

Reply:
I created an issue for this on Asana